### PR TITLE
Add a scraper to scrape Missoula City public meetings

### PR DESCRIPTION
**Pull request by smai-f**
work in progress

### CODE REVIEW

File: python/cdp_missoula_backend/scraper.py [hunk 1]
```diff
@@ -54,7 +58,29 @@ def get_events(
     # (Encoder1_AF_2022-03-02-02-03.mp4)
     # The video link is https://video.isilive.ca/missoula/ + value from previous step
 
+    options = webdriver.ChromeOptions()
+    options.add_argument("--ignore-certificate-errors")
+    options.add_argument("--incognito")
+    options.add_argument("--headless")
+    driver = webdriver.Chrome("./chromedriver", options=options)
+
+    msla_url = "https://pub-missoula.escribemeetings.com/?Year=2022"
```
Comment (smai-f) [inline python/cdp_missoula_backend/scraper.py]: TODO: Remove `?Year=2022`, the query param isn't retained when the driver makes the request, which is why line 71 is necessary

File: python/cdp_missoula_backend/scraper.py [hunk 2]
```diff
@@ -8,10 +8,75 @@
...
+                    )
+                    _meeting_data["title"] = title.text
+
+                    date = meeting.find_element(
+                        by=By.XPATH, value=".//div[@class='meeting-date']"
+                    )
+                    _meeting_data["date"] = date.text
+                    meetings_info.append(_meeting_data)
+                
+    return [len(meetings_info), meetings_info]
```
Comment (smai-f) [inline python/cdp_missoula_backend/scraper.py]: TODO: Verify `len(meetings_info)` 10 times and throw an exception if it exceeds some wild number

File: python/cdp_missoula_backend/scraper.py [hunk 3]
```diff
@@ -1,13 +1,82 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-from datetime import datetime
-from typing import List
+import time
```
Comment (smai-f) [inline python/cdp_missoula_backend/scraper.py:5]: if only

**Comment by smai-f** [comment]
woot

[98, [{'title': 'Administration and Finance Committee', 'date': 'Wednesday, 2 March 2022 @ 12:10 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_AF_2022-03-02-02-03.mp4'}, {'title': 'Administration and Finance Committee', 'date': 'Wednesday, 23 February 2022 @ 12:50 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_AF_2022-02-23-02-50.mp4'}, {'title': 'Administration and Finance Committee', 'date': 'Wednesday, 9 February 2022 @ 10:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_AF_2022-02-09-11-55.mp4'}, {'title': 'Administration and Finance Committee', 'date': 'Wednesday, 26 January 2022 @ 9:40 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_AF_2022-01-26-11-35.mp4'}, {'title': 'Administration and Finance Committee', 'date': 'Wednesday, 5 January 2022 @ 10:05 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_AF_2022-01-05-11-55.mp4'}, {'title': 'Affordable Housing Resident Oversight Committee', 'date': 'Wednesday, 9 March 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_AHROC_2022-03-09-07-51.mp4'}, {'title': 'Affordable Housing Resident Oversight Committee', 'date': 'Wednesday, 9 February 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_AHROC_2022-02-09-12-28.mp4'}, {'title': 'Affordable Housing Resident Oversight Committee', 'date': 'Wednesday, 12 January 2022 @ 6:00 PM'}, {'title': 'Bicycle and Pedestrian Advisory Board', 'date': 'Monday, 14 March 2022 @ 6:15 PM'}, {'title': 'Bicycle and Pedestrian Advisory Board', 'date': 'Monday, 14 February 2022 @ 6:15 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_BPAB_2022-02-14-08-12.mp4'}, {'title': 'Bicycle and Pedestrian Advisory Board', 'date': 'Monday, 10 January 2022 @ 6:15 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_BPAB_2022-01-10-08-08.mp4'}, {'title': 'Cemetery Board', 'date': 'Thursday, 3 February 2022 @ 12:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_CEM_2022-02-03-01-54.mp4'}, {'title': 'Cemetery Board', 'date': 'Thursday, 6 January 2022 @ 12:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_CEM_2022-01-06-01-51.mp4'}, {'title': 'City Council Meeting', 'date': 'Monday, 14 March 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_CCM_2022-03-14-07-50.mp4'}, {'title': 'City Council Meeting', 'date': 'Monday, 7 March 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_CCM_2022-03-07-07-53.mp4'}, {'title': 'City Council Meeting', 'date': 'Monday, 28 February 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_CCM_2022-02-28-07-57.mp4'}, {'title': 'City Council Meeting', 'date': 'Monday, 14 February 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_CCM_2022-02-14-07-52.mp4'}, {'title': 'City Council Meeting', 'date': 'Monday, 7 February 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_CCM_2022-02-07-07-53.mp4'}, {'title': 'City Council Meeting', 'date': 'Monday, 24 January 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_CCM_2022-01-24-07-48.mp4'}, {'title': 'City Council Meeting', 'date': 'Monday, 10 January 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_CCM_2022-01-10-07-49.mp4'}, {'title': 'City Council Meeting', 'date': 'Monday, 3 January 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_CCM_2022-01-03-07-58.mp4'}, {'title': 'City Council--No Meeting', 'date': 'Monday, 21 March 2022 @ 6:00 PM'}, {'title': 'City Council--No Meeting', 'date': 'Monday, 21 February 2022 @ 6:00 PM'}, {'title': 'City Council--No Meeting', 'date': 'Monday, 31 January 2022 @ 6:00 PM'}, {'title': 'City Council--No Meeting', 'date': 'Monday, 17 January 2022 @ 6:00 PM'}, {'title': 'Committee of the Whole', 'date': 'Wednesday, 16 March 2022 @ 10:20 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_COW_2022-03-16-12-15.mp4'}, {'title': 'Committee of the Whole', 'date': 'Wednesday, 23 February 2022 @ 10:10 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_Committee of the Whole_2022-02-23-12-06.mp4'}, {'title': 'Committee of the Whole', 'date': 'Wednesday, 5 January 2022 @ 9:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_COW_2022-01-05-10-54.mp4'}, {'title': 'Community Forum', 'date': 'Thursday, 24 February 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_Community Forum_2022-02-24-07-42.mp4'}, {'title': 'Community Forum', 'date': 'Thursday, 27 January 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_CF_2022-01-27-07-50.mp4'}, {'title': 'Conservation Lands Advisory Committee', 'date': 'Monday, 7 March 2022 @ 4:00 PM'}, {'title': 'Conservation Lands Advisory Committee', 'date': 'Monday, 7 February 2022 @ 4:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_CLAC_2022-02-07-05-54.mp4'}, {'title': 'Conservation Lands Advisory Committee', 'date': 'Monday, 10 January 2022 @ 3:30 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_CLAC_2022-01-10-05-19.mp4'}, {'title': 'Design Review Board', 'date': 'Wednesday, 9 March 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_DRB_2022-03-09-07-59.mp4'}, {'title': 'Design Review Board', 'date': 'Wednesday, 9 February 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_DRB_2022-02-09-08-15.mp4'}, {'title': 'Design Review Board', 'date': 'Wednesday, 12 January 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_DRB_2022-01-12-07-44.mp4'}, {'title': 'Energy and Climate Team', 'date': 'Thursday, 10 March 2022 @ 8:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_EC_2022-03-10-09-58.mp4'}, {'title': 'Energy and Climate Team', 'date': 'Thursday, 10 February 2022 @ 8:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_Energy and Climate Team_2022-02-10-09-57.mp4'}, {'title': 'Energy and Climate Team', 'date': 'Thursday, 13 January 2022 @ 8:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_EC_2022-01-13-09-59.mp4'}, {'title': 'Historic Preservation Commission', 'date': 'Wednesday, 2 February 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_HPC_2022-02-02-07-44.mp4'}, {'title': 'Historic Preservation Commission Subcommittee', 'date': 'Thursday, 24 February 2022 @ 4:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_Historic Preservation Commission Subcommittee_2022-02-24-06-07.mp4'}, {'title': 'Historic Preservation Commission Subcommittee', 'date': 'Wednesday, 2 February 2022 @ 3:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_Historic Preservation Commission Subcommittee_2022-02-02-04-49.mp4'}, {'title': 'Impact Fee Advisory Committee', 'date': 'Thursday, 27 January 2022 @ 10:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_IFAC_2022-01-27-11-56.mp4'}, {'title': 'Land Use and Planning Committee', 'date': 'Wednesday, 9 March 2022 @ 12:10 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_LUP_2022-03-09-02-18.mp4'}, {'title': 'Land Use and Planning Committee', 'date': 'Wednesday, 2 March 2022 @ 12:55 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_LUP_2022-03-02-02-54.mp4'}, {'title': 'Land Use and Planning Committee', 'date': 'Wednesday, 9 February 2022 @ 2:05 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_LUP_2022-02-09-04-06.mp4'}, {'title': 'Land Use and Planning Committee', 'date': 'Wednesday, 12 January 2022 @ 10:15 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_LUP_2022-01-12-12-14.mp4'}, {'title': 'Land Use and Planning Committee', 'date': 'Wednesday, 5 January 2022 @ 11:35 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_LUP_2022-01-05-01-34.mp4'}, {'title': "Mayor's Downtown Advisory Commission", 'date': 'Thursday, 10 February 2022 @ 2:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_MDAC_2022-02-10-03-37.mp4'}, {'title': "Mayor's Downtown Advisory Commission", 'date': 'Tuesday, 25 January 2022 @ 1:30 PM'}, {'title': 'Missoula Civic Television Advisory Commission', 'date': 'Monday, 14 February 2022 @ 3:30 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_MCTAC_2022-02-14-05-11.mp4'}, {'title': 'Missoula Civic Television Advisory Commission', 'date': 'Monday, 24 January 2022 @ 2:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_MCTAC_2022-01-24-03-58.mp4'}, {'title': 'Missoula Parking Commission', 'date': 'Tuesday, 8 February 2022 @ 12:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_MPC_2022-02-08-01-58.mp4'}, {'title': 'Missoula Parking Commission', 'date': 'Tuesday, 11 January 2022 @ 12:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_MPC_2022-01-11-01-56.mp4'}, {'title': 'Missoula Redevelopment Agency Board', 'date': 'Thursday, 17 March 2022 @ 12:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_MRA_2022-03-17-01-59.mp4'}, {'title': 'Missoula Redevelopment Agency Board', 'date': 'Thursday, 17 February 2022 @ 12:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_MRA_2022-02-17-01-59.mp4'}, {'title': 'Missoula Redevelopment Agency Board', 'date': 'Thursday, 20 January 2022 @ 12:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_MRA_2022-01-20-01-59.mp4'}, {'title': 'Open Space Advisory Committee', 'date': 'Thursday, 10 March 2022 @ 4:00 PM'}, {'title': 'Open Space Advisory Committee', 'date': 'Thursday, 10 February 2022 @ 4:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_OSAC_2022-02-10-05-50.mp4'}, {'title': 'Parks and Conservation Committee', 'date': 'Wednesday, 16 March 2022 @ 11:30 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PC_2022-03-16-01-32.mp4'}, {'title': 'Parks and Conservation Committee', 'date': 'Wednesday, 9 March 2022 @ 10:30 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder 1_PC_2022-03-09-1-42-40.mp4'}, {'title': 'Parks and Conservation Committee', 'date': 'Wednesday, 2 March 2022 @ 9:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PC_2022-03-02-10-52.mp4'}, {'title': 'Parks and Conservation Committee', 'date': 'Wednesday, 23 February 2022 @ 12:20 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PC_2022-02-23-02-17.mp4'}, {'title': 'Parks and Conservation Committee', 'date': 'Wednesday, 9 February 2022 @ 1:20 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PC_2022-02-09-03-14.mp4'}, {'title': 'Parks and Conservation Committee', 'date': 'Wednesday, 26 January 2022 @ 11:55 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_Parks and Conservation Committee_2022-01-26-01-54.mp4'}, {'title': 'Parks and Conservation Committee', 'date': 'Wednesday, 5 January 2022 @ 1:30 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PC_2022-01-05-03-24.mp4'}, {'title': 'Parks and Recreation Board', 'date': 'Tuesday, 8 March 2022 @ 12:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_PRB_2022-03-08-01-59.mp4'}, {'title': 'Parks and Recreation Board', 'date': 'Tuesday, 8 February 2022 @ 12:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_PRB_2022-02-08-02-00.mp4'}, {'title': 'Parks and Recreation Board', 'date': 'Tuesday, 11 January 2022 @ 12:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_PRB_2022-01-11-01-58.mp4'}, {'title': 'Planning Board', 'date': 'Tuesday, 22 March 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PB_2022-03-22-07-58.mp4'}, {'title': 'Planning Board', 'date': 'Tuesday, 15 March 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PB_2022-03-01-07-55.mp4'}, {'title': 'Planning Board', 'date': 'Tuesday, 1 March 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PB_2022-03-01-08-04.mp4'}, {'title': 'Planning Board', 'date': 'Tuesday, 15 February 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PB_2022-02-15-07-56.mp4'}, {'title': 'Planning Board', 'date': 'Tuesday, 1 February 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PB_2022-02-01-07-58.mp4'}, {'title': 'Planning Board', 'date': 'Tuesday, 18 January 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_Planning Board_2022-01-18-07-58.mp4'}, {'title': 'Planning Board', 'date': 'Tuesday, 4 January 2022 @ 6:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PB_2022-01-04-07-59.mp4'}, {'title': 'Police Commission', 'date': 'Tuesday, 8 February 2022 @ 3:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_POL_2022-02-08-04-49.mp4'}, {'title': 'Public Art Committee', 'date': 'Tuesday, 15 March 2022 @ 4:00 PM'}, {'title': 'Public Art Committee', 'date': 'Tuesday, 15 February 2022 @ 4:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/live2'}, {'title': 'Public Art Committee', 'date': 'Tuesday, 18 January 2022 @ 4:00 PM', 'video_uri': 'https://video.isilive.ca/missoula/live2'}, {'title': 'Public Safety and Health Committee', 'date': 'Wednesday, 2 March 2022 @ 10:15 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PSH_2022-03-02-12-16.mp4'}, {'title': 'Public Safety and Health Committee', 'date': 'Wednesday, 26 January 2022 @ 11:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PSH_2022-01-26-01-04.mp4'}, {'title': 'Public Safety and Health Committee', 'date': 'Wednesday, 5 January 2022 @ 12:20 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PSH_2022-01-05-02-21.mp4'}, {'title': 'Public Works Committee', 'date': 'Wednesday, 16 March 2022 @ 1:15 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PW_2022-03-16-03-12.mp4'}, {'title': 'Public Works Committee', 'date': 'Wednesday, 9 March 2022 @ 9:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PW_2022-03-09-10-55.mp4'}, {'title': 'Public Works Committee', 'date': 'Wednesday, 2 March 2022 @ 10:40 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PW_2022-03-02-12-35.mp4'}, {'title': 'Public Works Committee', 'date': 'Wednesday, 23 February 2022 @ 11:10 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PW_2022-02-23-01-03.mp4'}, {'title': 'Public Works Committee', 'date': 'Wednesday, 9 February 2022 @ 9:15 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PW_2022-02-09-11-05.mp4'}, {'title': 'Public Works Committee', 'date': 'Wednesday, 26 January 2022 @ 1:50 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PW_2022-01-26-03-43.mp4'}, {'title': 'Public Works Committee', 'date': 'Wednesday, 19 January 2022 @ 9:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PW_2022-01-19-10-52.mp4'}, {'title': 'Public Works Committee', 'date': 'Wednesday, 12 January 2022 @ 9:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PW_2022-01-12-11-05.mp4'}, {'title': 'Public Works Committee', 'date': 'Wednesday, 5 January 2022 @ 12:55 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder1_PW_2022-01-05-02-51.mp4'}, {'title': 'Transportation Policy Coordinating Committee', 'date': 'Tuesday, 15 March 2022 @ 1:30 PM'}, {'title': 'Transportation Policy Coordinating Committee', 'date': 'Tuesday, 15 February 2022 @ 1:30 PM'}, {'title': 'Transportation Policy Coordinating Committee', 'date': 'Tuesday, 18 January 2022 @ 1:30 PM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_TPCC_2022-01-18-03-25.mp4'}, {'title': 'Transportation Technical Advisory Committee', 'date': 'Thursday, 3 March 2022 @ 10:00 AM'}, {'title': 'Transportation Technical Advisory Committee', 'date': 'Thursday, 3 February 2022 @ 10:00 AM', 'video_uri': 'https://video.isilive.ca/missoula/Encoder2_TTAC_2022-02-03-11-50.mp4'}, {'title': 'Transportation Technical Advisory Committee', 'date': 'Thursday, 6 January 2022 @ 10:00 AM'}]]

**Comment by smai-f** [comment]
TODO: Remove non-video-ed meetings

File: python/cdp_missoula_backend/scraper.py [hunk 4]
```diff
@@ -69,38 +88,95 @@ def get_scraped_data() -> List:
...
+                )
+            except:
+                # it's likely this video is corrupt, but give peace a chance to see if
+                # the file_name is present
+                pass
+
             player = driver.find_element(by=By.XPATH, value="//div[@id='isi_player']")
             file_name = player.get_attribute("data-file_name")
+            if file_name.count(".mp4") == 0:
+                print(f"Erroring out on file_name {file_name}")
```
Comment (smai-f) [inline python/cdp_missoula_backend/scraper.py]: It would be fun to auto-email Missoula City and let them know their video link is broken

Comment (smai-f) [reply python/cdp_missoula_backend/scraper.py]: Erroring out on file_name live2
{'date': datetime.datetime(2022, 2, 15, 16, 0), 'video_player_uri': 'https://pub-missoula.escribemeetings.com/Players/ISIStandAlonePlayer.aspx?Id=ecf3882b-25dd-4d9b-9b1c-2ca993e94a10', 'title': 'Public Art Committee'}
Erroring out on file_name live2
{'date': datetime.datetime(2022, 1, 18, 16, 0), 'video_player_uri': 'https://pub-missoula.escribemeetings.com/Players/ISIStandAlonePlayer.aspx?Id=9b84a1c5-81a3-4999-8150-219159b2a1b2', 'title': 'Public Art Committee'}

**Comment by smai-f** [comment]
@JacksonMaxfield some info on Missoula public meetings:

TOTAL MEETING TIME SINCE JAN 1, 2022
4 days, 19:08:17
~$72 initial startup cost to run speech to text

AVERAGE MEETING TIME PER WEEK
9:35:41.416667
ongoing ~$6 a week cost to run speech to text

**Comment by evamaxfield** [comment]
> @JacksonMaxfield some info on Missoula public meetings:
> 
> TOTAL MEETING TIME SINCE JAN 1, 2022 4 days, 19:08:17 ~$72 initial startup cost to run speech to text
> 
> AVERAGE MEETING TIME PER WEEK 9:35:41.416667 ongoing ~$6 a week cost to run speech to text

@smai-f thanks for the info. Will add to the cookiecutter README if that is okay?
(and that cost is _okay_? / makes sense?)

**Comment by smai-f** [comment]
@JacksonMaxfield I'm about to run this thing for real overnight tonight, and will get you the real cost!

**Comment by smai-f** [comment]
I'm merging this as-is with the hardcoded event returned to test the output in CI

**Pull request merged**